lib/digital_ocean/ssh_keys.ex

```diff
@@ -0,0 +1,96 @@
+defmodule DigitalOcean.SSHKeys do
+  alias DigitalOcean.{ Operation }
+
+  @doc """
+  Create a new SSH key.
+
+  ## Example creating a new SSH key
+
+      iex> DigitalOcean.SSHKeys.create(
+      ...>   name: "my ssh public key",
+      ...>   public_key: "ssh-rsa AEXAMPLEaC1yc2EAAAADAQABAAAAQQDDHr/jh2Jy4yALcK4JyWbVkPRaWmhck3IgCoeOO3z1e2dBowLh64QAM+Qb72pxekALga2oi4GvT+TlWNhzPH4V example",
+      ...> ) |> DigitalOcean.request()
+      { :ok, %DigitalOcean.Response{} }
+  """
+  @type create_opts :: [name: String.t(), public_key: String.t()]
```

Comment:
We don't want to be responsible for the arguments sent to the API. If they change then we'll cause dialyzer errors for developers using this library that use arguments not present in the type. So please remove.